parsl/executors/high_throughput/interchange.py

```diff
@@ -89,8 +89,9 @@ def __init__(self,
         client_address : str
              The ip address at which the parsl client can be reached. Default: "127.0.0.1"
 
-        interchange_address : str
```

Comment:
I'm trying to understand how this parameter was used previously: `git grep` suggests it was not used at all? Is that right?

Reply:
It was not used at all, but I believe this is how it was meant to be used.

Reply:
then maybe can use a single address object attribute instead of both `self.interchange_address` and `self.listen_address` ?